profiler/src/TraceAnalyzer.php

```
@@ -111,33 +111,33 @@ private function parseTraceFile($handle): void
             $this->analysisResult['📈 statistics']['🏗️ max_call_depth'] = max(
                 $this->analysisResult['📈 statistics']['🏗️ max_call_depth'],
-                $level
+                $level,
             );
 
             if ($type === '0' && count($parts) >= 7) {
                 // Function entry
                 $functionName = $parts[5] ?? '';
-                $userDefined = (int)($parts[6] ?? 0);
+                $userDefined = (int) ($parts[6] ?? 0);
                 $includeFilename = $parts[7] ?? '';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$includeFilename'. (undefined)

(UnusedLocalVariable)